tests/TestSoftDeletableModelFactory.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Tests;
+
+use Carbon\Carbon;
+use Tests\Models\TestSoftDeletableModel;
+
+class TestSoftDeletableModelFactory
```

Comment:
Can you let this factory extend the other factory? This way a lot of code here can be removed.

Reply:
done